build_files_updated_unittest.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This script verifies that BUILD files and cmake files are in sync with src/Makefile.am
```

Comment:
I think you may need to add `set -e` at the top of this test script so that a failing command causes the script to return a non-zero error code. Otherwise one of the `diff` commands could detect a difference without the test failing.

Reply:
Done!